utopia-remix/app/routes/projects.tsx

```diff
@@ -698,18 +679,12 @@ const ProjectCard = React.memo(
       }
     }, [openShareDialog, canOpenShareDialog])
 
-    const onMouseDown = React.useCallback(
-      (event: React.MouseEvent) => {
-        // on right click only allow selection (not de-selction)
-        if (event.button !== 2 || !selected) {
-          onSelect()
-        }
-      },
-      [onSelect, selected],
-    )
+    const toggleContextMenuOpen = React.useCallback((isOpen: boolean) => {
```

Comment:
this doesn't look like a toggle to me!

Reply:
Hmm, I've always thought of "toggle" as also "toggle on" or "toggle off" and not just "change to the opposite of the current state"